BurnerB/FreeMentors-V2
# [#168093218] Get all sessions

#### What does this PR do?
Allows a mentor or user  to view all sessions
#### Description of Task to be completed?
* For a ​ user :The response should be a list of all mentorship
request sessions created by the user
*  For a ​ mentor​ :The response should be a list of all mentorship
request sessions created against the mentor.
* Tests should pass
* Refractor tests
#### How should this be manually tested?
* Clone this  repository https://github.com/BurnerB/FreeMentors-V2.git and cd into FreeMentors-V2
* checkout to branch ft-get-all-sessions-168093218
* Run npm --init to install all dependancies.
* Run command npm test on your terminal
* Tests should pass
* Open postman and GET `/sessions`
 * Should recieve response:
 ```
 {
   "status": 200,
    "data":
    [
        {
        sessionId:1
        mentorId:1
        menteeId:1
        questions:"a question"
        menteeEmail:"email"
        status:"accepted or declined"
            
        },
        ]
}
 ```
#### What are the relevant pivotal tracker stories?
[#168093218](https://www.pivotaltracker.com/story/show/168093218)
#### Screenshots (if appropriate) 

![PASSING GET ALL SESSIONS](https://user-images.githubusercontent.com/45785786/63640451-2c81fd80-c6a9-11e9-8332-cd31177edf78.png)
